.gitignore

```diff
@@ -6,6 +6,7 @@
 /pkg/
 /spec/reports/
 /tmp/
+/Gemfile.lock
```

Comment:
i wasn't sure about this, but this comment:
https://github.com/radar/guides/issues/14#issuecomment-328864865

succinctly describes why gemfile.lock shouldn't be used for gems:

> Since the gem command only respects the dependency version constraints named in the gemspec, not Gemfile.lock, the presence of a Gemfile.lock is more likely to keep gem maintainers unaware of incompatibility with newer versions of dependencies that gem users will install but maintainers will not.

Reply:
Exactly. The most referenced article about this is https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/